### ISSUE TITLE
Redirect most blocked domains to local webserver with proper block page

### ISSUE THREAD

**Issue by Bluscream** [AdguardTeam/AdGuardHome#811]
- [ ] Redirect all blocked requests that don't have `$drop` rule to a local webserver which serves a proper "Blocked by AdguardHome" page with "unblock" and "unblock this time" buttons
<hr>

![image](https://user-images.githubusercontent.com/3318223/59064548-d02def80-88aa-11e9-903d-a7378bbb2fca.png)

<hr>

**Comment by ameshkov** [comment]
Isn't it a duplicate of #454 ?
